vespaclient-container-plugin/src/test/java/com/yahoo/document/restapi/resource/MockedOperationHandler.java

```diff
@@ -18,9 +18,11 @@
     int deleteCount = 0;
 
     @Override
-    public VisitResult visit(RestUri restUri, String documentSelection, Optional<String> cluster, Optional<String> continuation) throws RestApiException {
-        return new VisitResult(Optional.of("token"), "List of json docs, cont token " + continuation.map(a->a).orElse("not set") + ", doc selection: '"
-                + documentSelection + "'");
+    public VisitResult visit(RestUri restUri, String documentSelection, VisitOptions options) throws RestApiException {
+        return new VisitResult(Optional.of("token"), "List of json docs, cont token "
+                + options.continuation.map(a->a).orElse("not set") + ", doc selection: '"
```

Comment:
What's the point of this `.map()` ?

Reply:
Redundant identity transform. I'll remove it.